docs/how-to-guides/tracing/index.md

```diff
@@ -117,4 +117,14 @@ pf run create --flow . --data "./data/bert-paper-qna.jsonl" --column-mapping cha
 ```
 Then you will get a run related trace URL, e.g. http://localhost:<port>/v1.0/ui/traces?run=chat_with_pdf_20240226_181222_219335
 
-![batch_run_record](../../media/trace/batch_run_record.png)
\ No newline at end of file
+![batch_run_record](../../media/trace/batch_run_record.png)
+
+### Search
```

Comment:
need to add change log, so user know this is added in 1.10

Reply:
i will prepare a separate pr for changelog update for this